backend/src/services/sessionExec.js

```diff
@@ -360,11 +361,11 @@ const addConfusionTS = async ({ sessionId, difficulty, speed }) => {
   // extract the saved confusion timestep and convert it to a plain object
   // then readd the mongo _id field under the id key and publish the result
   // this is needed as redis swallows the _id field and the client could break!
-  // const savedConfusion = session.confusionTS[session.confusionTS.length - 1].toObject()
-  // pubsub.publish(CONFUSION_ADDED, {
-  //   [CONFUSION_ADDED]: { ...savedConfusion, id: savedConfusion._id },
-  //   sessionId,
-  // })
+  const savedConfusion = session.confusionTS[session.confusionTS.length - 1].toObject()
```

Comment:
this subscription is not currently used. here, the backend pushes a new confusionTS as incoming directly to the subscription on the client. however, as we have commented out `subscribeToMore` on the client, the subscription is never created, and these events disappear in nirvana.

at some point, we should probably reintroduce a working subscription for the confusion barometer, so the lecturer gets a nice real-time view on the gauge.

Reply:
probably we should move the average computation to the backend ahead of release anyways. Maybe we can then combine this step with reintroducing the subscribeToMore